src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/navigation/mobileNav/local/CommunityBarMobile.java

```diff
@@ -1,4 +1,4 @@
-package com.wikia.webdriver.pageobjectsfactory.componentobject.navigation.local;
+package com.wikia.webdriver.pageobjectsfactory.componentobject.navigation.mobileNav.local;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00120)